src/java.base/share/classes/sun/nio/ch/DirectBuffer.java

```diff
@@ -30,6 +30,13 @@
 
 public interface DirectBuffer {
 
+    // Use of the returned address must be guarded if this DirectBuffer
+    // is backed by a memory session that is explicitly closeable.
+    //
+    // Failure to do this means the outcome is undefined including
+    // silent unrelated memory mutation and JVM crashes.
+    //
+    // Guards are available in the JavaNioAccess class.
```

Comment:
Did you mean to leave the word "Guards" in the comment? I guess I would say something "See JavaNioAccess for methods to acquire/release" or something like that.